biblateka/main_branch_xatosini_tugirlash.md

```diff
@@ -0,0 +1,53 @@
+#### Main branchdagi xatoni to'g'irlash
+Vazifa shundan iboratki biz main branchdan biror xatoni to'girlamoqchimiz (yoki main branchga o'zgartirish kiritmoqchimiz). <br>
+Buning uchun qanday ishlarni amalga oshiramiz.
+
+1. Yangi branch ochib olamiz. 
+    ```codeline
+    git checkout -b 10-kun_xatosi
+    ```
+   Bu buyruq orqali 10-kun_xatosi nomli yangi branch hosil qilib oldik. <br>
```

Comment:
Bu gapni 5-qatorga qo'shish kerak. Keyin, O'sha kommanda nafaqat branch ochadi balki o'shanga o'tadi hamligni aytib o'tish kerak.

Reply:
To'g'irlandi